README.md

```diff
@@ -186,29 +194,26 @@ Slaves to bridge/bond/team devices cannot specify a zone.
 ```yaml
 network_connections:
   - name: "eth0"
-    #state: up        # implicit default, as there is no type specified
-    wait: 0
+    state: up
 ```
 
-The above example defaults to `state=up` and requires an existing profile to activate.
-Note that if neither `type` nor `state` is specifed, `up` is implied. Thus in above
-example the `state` is redundant.
+The above example requires an existing profile to activate.
 
 - For NetworkManager this results in `nmcli connection id {{name}} up`.
 
 - For initscripts it is the same as `ifup {{name}}`.
 
-`up` also supports an optional integer argument `wait`. `wait=0` will only initiate
-the activation but not wait until the device is fully connected. Connection will complete
-in the background, for example after a DHCP lease was received.
-`wait: <SECONDS>` is a timeout for how long we give the device
-to activate. The default is `wait=-1` which uses a suitable timeout. Note that this
-argument only makes sense for NetworkManager.
+State `up` also supports an optional integer argument `wait`. `wait=0` will
+only initiate the activation but not wait until the device is fully connected.
+Connection will complete in the background, for example after a DHCP lease was
+received. `wait: <SECONDS>` is a timeout for how long we give the device to
+activate. The default is `wait=-1` which uses a suitable timeout. Note that
+this argument only makes sense for NetworkManager.
 **TODO** `wait` different from zero is not yet implemented.
```

Comment:
`wait` different from zero is not yet implemented. - but the default is -1, is this not implemented?

Reply:
From what I understand, it does not seem to be possibly to specify `wait:-1`. Since there is no change for the behavior in the code and I just fixed the wording here we should probably investigate this later.

Reply:
I filed #79 for this.

Reply:
>  it does not seem to be possibly to specify wait:-1

Can we then write "The default is to use a suitable timeout." without referring to the special -1 value?